From 7e7308158c9149c33309c0d36a6e1126e690fb58 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 2 Mar 2011 11:06:49 +0100
Subject: [PATCH] fips: do not load tcrypt with "noexit" parameter

"noexit=1" is the default mode for the tcrypt module now.
---
 modules.d/01fips/fips.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/01fips/fips.sh b/modules.d/01fips/fips.sh
index 2f6cb97..e0480cb 100755
--- a/modules.d/01fips/fips.sh
+++ b/modules.d/01fips/fips.sh
@@ -52,7 +52,7 @@ do_fips()
         fi
     done
     info "Self testing crypto algorithms"
-    modprobe tcrypt noexit=1 || return 1
+    modprobe tcrypt || return 1
     rmmod tcrypt
     info "All initrd crypto checks done"  
 
